src/nebula/NebulaContext.cs

```diff
@@ -25,12 +26,28 @@ public NebulaContext()
 
         internal IComponentContext ComponentContext { get; set; }
 
+        public string MongoConnectionString { get; set; }
```

Comment:
These connection strings are component-specific. I mean, they only have meaning when using Mongo as a store and for Redis queues as step sources. We need to apply some king of a generalization later, and deprecate these properties.